examples/sht31_test.py

```diff
@@ -0,0 +1,29 @@
+import time
+import board
+import busio
+import adafruit_sht31
+
+# Create library object using our Bus I2C port
+i2c = busio.I2C(board.SCL, board.SDA)
+sensor = adafruit_sht31.SHT31(i2c)
+
+
+while True:
+    print(sensor.temperature_and_relative_humidity)
+    print(sensor.temperature)
+    print(sensor.relative_humidity)
+    print(sensor.status)
+    print(sensor.heater)
+    sensor.heater=True
+    print(sensor.heater)
+    print(sensor.status)
+    sensor.heater=False
+    print(sensor.status)
+    sensor.heater=True
+    print(sensor.heater)
+    print(sensor.status)
+    sensor.reset
```

Comment:
Hmm `sensor.reset` seems weird like this, make it a function if you don't actually assign anything?

Reply:
changed to a function   - I did not include this in sht31_simpletest. Not sure if anyone will actually use it.